test/conv2d_bwd_data/conv2d_bwd_data.cpp

```diff
@@ -302,19 +304,19 @@ int main(int argc, char* argv[])
 
     if(data_type == 0)
     {
-        Run(F32(), F32(), F32());
+        Run(F32(), F32(), F32(), F32());
```

Comment:
If Run() fail, what will be the return value of main()?

Reply:
done